common/disable-arrow-scroll.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
This file name does not seem to follow naming conventions for the web, probably it should be renamed to `disable_arrow_scroll.js`

Is this file really required as a separate file or maybe the logic can be moved to `ui_manager.js`?

Reply:
This is not my change, or file. This is one of the changes from someone else's commit I added it to fix the merge conflict. This came from your end when you shared the conflicts. I can move the logic to my file if you want me to though.